projects/fusion-ui/src/lib/components/ui-components/dropdown/multi-dropdown/multi-dropdown.component.ts

```diff
@@ -149,11 +149,10 @@ export class MultiDropdownComponent extends DropdownComponent implements OnInit,
                 }
             });
         }
+        this.isInSelectAllAction();
```

Comment:
why you've changed multidropdown component?

Reply:
It fix for select all bug in multi-dropdown without confirmation (footer).